package.json

```diff
@@ -16,26 +16,27 @@
     "LICENSE"
   ],
   "scripts": {
-    "test": "jshint lib index.js && jscs lib index.js && mocha -t 5000 -b -R spec test/index",
-    "cover": "nyc --reporter=lcov --reporter=text-summary npm test"
+    "lint": "eslint .",
+    "test": "jshint lib index.js && mocha -t 5000 -b -R spec test/index",
+    "cover": "eslint . && nyc --reporter=lcov --reporter=text-summary npm test"
   },
   "dependencies": {
     "extend": "^3.0.0",
-    "findup-sync": "^2.0.0",
+    "findup-sync": "^3.0.0",
```

Comment:
major bump probably requires a major bump from us?  I forget why I major bumped that module.

Reply:
It seems there is no breaking changes. Updating is-glob and normalize-path.
I think this bump is good.